rasa/core/processor.py

```diff
@@ -93,19 +93,23 @@ def __init__(
         self.max_number_of_predictions = max_number_of_predictions
         self.on_circuit_break = on_circuit_break
         self.action_endpoint = action_endpoint
+        self.model_filename = ""
         self.model_metadata, self.graph_runner = self._load_model(model_path)
```

Comment:
nit: how about we let the load return the model name just like the other of the loaded assets? (then setting the filename is not so much an implicit side effect)
```suggestion
        self.model_filename, self.model_metadata, self.graph_runner = self._load_model(model_path)
```

Reply:
Done 👍